yb-voyager/cmd/analyzeSchema.go

```diff
@@ -259,6 +260,29 @@ func reportSummary() {
 	}
 }
 
+func addSummaryDetailsForIndexes() {
+	var indexesInfo []utils.IndexInfo
+	found, err := metaDB.GetJsonObject(nil, metadb.INDEXES_INFO_KEY, &indexesInfo)
+	if err != nil {
+		utils.ErrExit("analyze schema report summary: load indexes info: %s", err)
+	}
+	if found {
+		indexesPresent := lo.Keys(summaryMap["INDEX"].objSet)
+		unexportedIdxsMsg := "Indexes which are neither exported by yb-voyager as they are unsupported in YB and needs to be handled manually:\n"
+		unexportedIdxsPresent := 0
+		for _, indexInfo := range indexesInfo {
+			exportedIdxName := indexInfo.TableName + "_" + strings.Join(indexInfo.Columns, "_")
+			if !slices.Contains(indexesPresent, strings.ToLower(exportedIdxName)) {
```

Comment:
I suggest: 
rename `indexesPresent` -> `exportedIndexes`, `exportedIdxName` -> `sourceIdxName` 
My assumption is correct, right? indexes present in summaryMap are the ones which have been exported by export schema, and those present in indexesInfo json object are all the indexes in the source DB.

Reply:
right